### PR TITLE
[stable/postgresql] add master/slave labels/annotations

### PR DESCRIPTION
**Pull request by vbehar**
#### What this PR does / why we need it:

Add support for custom labels and annotations on the master & slave statefulset, so that people can enable third-party kubernetes features that use either labels or annotations on the statefulset itself, and not the pods.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/postgresql/values.yaml [hunk 1]
```diff
@@ -260,6 +260,8 @@ master:
   nodeSelector: {}
   affinity: {}
   tolerations: []
+  labels: {}
+  annotations: {}
```
Comment (juan131) [inline stable/postgresql/values.yaml:264]: Please update these parameters in the **values-production.yaml** too

File: stable/postgresql/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 6.3.12
+version: 6.3.13
```
Comment (juan131) [inline stable/postgresql/Chart.yaml]: Increase minor version instead: `6.4.0`

**Review by juan131 — CHANGES_REQUESTED** [review]
Hi @vbehar 

Thanks so much for this contribution! 

Could you please take a look to my suggestions when you have a chance?

File: stable/postgresql/Chart.yaml (hunk 2 above)
Comment (vbehar) [reply stable/postgresql/Chart.yaml]: thanks for the review. I increased the minor version, and also took the time to add labels/annotations on the slaves too, to be consistent

**Review by juan131 — APPROVED** [review]
/lgtm
/ok-to-test

**Comment by juan131** [comment]
Thanks so much for addressing the suggestions. I'm afraid other commits were landed since I reviewed this PR, could you please rebase from master and update the PR?

**Comment by vbehar** [comment]
@juan131 ok, rebased and pushed

**Review by juan131 — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**